sdks/java/testing/test-utils/src/main/java/org/apache/beam/sdk/testutils/publishing/InfluxDBPublisher.java

```diff
@@ -64,6 +64,13 @@ public static void publishNexmarkResults(
     publishWithCheck(settings, () -> publishNexmark(results, settings, tags));
   }
 
+  public static void publishTpcdsResults(
```

Comment:
Honestly, I'm not particularly happy to see more of these publisher interfaces added :/
In my eyes it's quite a bad sign that every new use case requires adding a new "interface" to `InfluxDBPublisher` (not to mention conversion of datatypes for InfluxDB scattered around the metric collection / test logic ). Just saying ....
Might be worth it building this on top of https://github.com/apache/beam/pull/22260 or use the InfluxDB java client instead.

Reply:
Of course we can create own publishers for every case but does it worth in this case?

Reply:
My point here is that the `InfluxDBPublisher` abstraction is so poor that you are forced to add a new interface method for every single new use case ... the PR linked above fixes that for the JMH benchmarks. But it might be even better to just use https://mvnrepository.com/artifact/com.influxdb/influxdb-client-java and not depend on these fairly broken `test-utils` at all

Reply:
Personally, I'd prefer to use only one API in Beam Java SDK and internal `InfluxDBPublisher ` with a fixed API should fit well. What is a status of #22260 ? Does it miss something or it can be merged? 

I'd prefer to merge this PR asap to start keep TPC-DS metrics in InfluxDB.

Reply:
https://github.com/apache/beam/pull/22260 requires a review, I can ping Luke to see if he's back or if you wanna do it that would be perfect as well...
Regarding one API .... IMHO I don't think that should be the goal, I'd consider it to be much more important to not build new things on clearly broken and poor old abstractions / code

Reply:
I totally agree to have a good API for such internal tools but if every perf test will use an own InfluxDB client implementation then it will be even worse, imho. So, we need to have an own API for that (as we may conclude basing on other parts of Beam that don't have...), though an internal implementation can vary during the time.

Reply:
I agree, every (perf) test should use the very same InfluxDB client implementation using ONE well defined interface. That's why this client CANNOT be the current client. Copy pasting the same code over and over again to create a somehow slightly different interface for the next test or creating a brand new client is both just as bad. 

I'm fixing that in https://github.com/apache/beam/pull/22260, but in the end it just ends up very close to what the offical Java InfluxDB client does. So why maintaining an "internal" API for InfluxDB at all? The official JAVA InfluxDB client does all you need and is trivial to understand. Which isn't the case with the current code! Figuring out how data ends up in InfluxDB is impossible from looking at the interfaces.

Reply:
I'm ok to have an own Beam API with official Java InfluxDB client inside but likely not expose this client as a public interface (as we do in other case and it may cause issues in a long term).